website/components/CodeBlock.js

```diff
@@ -69,8 +70,9 @@ export default function CodeBlock({
                 transform: 'scale(0.95, 0.95)',
               },
             }}
-            aria-label="Copy code">
```

Comment:
Having visually hidden content is usually better than `aria-label`, which should be used more as a last resort.